fetch_last_webpage.py

```diff
@@ -16,20 +16,16 @@ def fetch_last_webpage():
 
   # Close the cursor
   cursor.close()
-  print(result_string)
   return json.loads(result_string)
 
 
 def save_new_version(json_formatted):
     con = mydb.cursor()
     json_string = json.dumps(json_formatted)
-    con.execute("UPDATE HTMLFile SET WebsiteText=%s WHERE ID=%s",  (json_string, 1))
+    res = con.execute("UPDATE HTMLFile SET WebsiteText=(%s) WHERE ID=1", (json_string,))
```

Comment:
Did this work for you on your local computer? It looks fine, since it forces Python to recognize the tuple, but I didn't know the parentheses around the `%s` were necessary

Reply:
yea it fixed it